src/Tribe/RSVP.php

```diff
@@ -2836,4 +2836,17 @@ public function decrease_ticket_sales_by( $ticket_id, $quantity = 1 ) {
 
 		return $sales;
 	}
+
+	/**
+	 * Check if the not going option is enabled for the provided rsvp ticket id.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $ticket_id The ticket post ID.
+	 *
+	 * @return bool Whether the not going option is enabled or not.
+	 */
+	public function is_not_going_enabled( $ticket_id ) {
```

Comment:
```suggestion
	public function is_not_going_enabled( $ticket_id ): bool {
```

Should we cast the param as int or do a safe check before the return?